Ginger/Ginger/RunSetLib/CreateAutoRunWizardLib/AutoRunWizard.cs

```diff
@@ -54,57 +54,133 @@ public AutoRunWizard(RunSetConfig runSetConfig, Context context)
-                    Reporter.ToUser(eUserMsgKey.StaticInfoMessage, "Auto Run Configuration and Shortcut were created successfully.");
-                    return;
+                    userMsg += SaveShortcut();
+                }
+                if (AutoRunShortcut.StartExecution)
+                {
+                    userMsg = ExecuteCommand(userMsg);
+                }
+                if (!string.IsNullOrEmpty(userMsg))
+                {
+                    Reporter.ToUser(eUserMsgKey.RunsetAutoRunResult, "Please find auto run wizard outcomes:" + Environment.NewLine + userMsg);
                 }
-
-                Reporter.ToUser(eUserMsgKey.StaticInfoMessage, "Auto Run Configuration was created successfully.");
             }
             catch(Exception ex)
             {
-                Reporter.ToUser(eUserMsgKey.StaticErrorMessage, "Error occurred while creating the Auto Run Configuration/Shortcut." + Environment.NewLine + "Error: " + ex.Message);
+                Reporter.ToUser(eUserMsgKey.StaticErrorMessage, "Error occurred while creating the Auto Run Configuration/Shortcut/Execution." + Environment.NewLine + "Error: " + ex.Message);
             }
         }
 
-        public void SaveShortcut()
+        private string ExecuteCommand(string userMsg)
         {
-            WshShell shell = new WshShell();
-            IWshShortcut shortcut = (IWshShortcut)shell.CreateShortcut(AutoRunShortcut.ShortcutFileFullPath);
-            shortcut.Description = AutoRunShortcut.ShortcutFileName;
-            shortcut.WorkingDirectory = AutoRunShortcut.ExecuterFolderPath;
-            if (!String.IsNullOrEmpty(Environment.GetEnvironmentVariable("GINGER_HOME")))
-            {                                
-                shortcut.TargetPath = "ginger"; 
-                shortcut.Arguments = AutoRunConfiguration.ConfigArgs;
+            var count = 1;
+            var successCount = 0;
+            var failCount = 0;
+            while (count <= AutoRunConfiguration.ParallelExecutionCount)
+            {
+                try
+                {
+                    if (AutoRunConfiguration.IsRequestAPIExecution)
+                    {
+                        var responseString = new CLIRequestAPI().ExecuteFromRunsetShortCutWizard(AutoRunConfiguration.ExecutionServiceUrl, AutoRunConfiguration.CLIContent);
+
+                        if (responseString == "Created")
+                        {
+                            successCount++;
+                        }
+                        else
+                        {
+                            failCount++;
+                            userMsg += "Failed to start remote execution, error: " + responseString + Environment.NewLine;
+                        }
+                    }
+                    else
+                    {
+                        var args = AutoRunConfiguration.CLIContent;
+
+                        if (AutoRunConfiguration.SelectedCLI.Verb == "dynamic")
```

Comment:
let's not use strings comparison like in below, but enums
AutoRunConfiguration.SelectedCLI.Verb == "dynamic"